lib/datasource/npm/releases.ts

```diff
@@ -11,7 +11,7 @@ export async function getReleases({
   }
   const res = await getDependency(lookupName);
   if (res) {
-    res.tags = res['dist-tags'];
+    res.releaseTags = res['dist-tags'];
```

Comment:
What the cause for this?

Reply:
Purely opinionated - I thought `tags` was too generic. I can revert though